topology/University of Colorado/UColorado_HEP/UColorado_HEP.yaml

```
@@ -34,7 +34,7 @@ Resources:
       GridFtp:
         Description: GridFtp Storage Element
         Details:
-          hidden: false
+          hidden: false   
```

Removing an extra whitespace change

```suggestion
          hidden: false
```